### PR TITLE
Clarified a bit more documentation, fixed a lot of typos and removed some redundant codes.

### PR DESCRIPTION
**Pull request by heavygabriel**
I also changed the copyright year.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,118 +1,118 @@
...
 
 v0.1.8
 ======
-* Fixes on some float to int x87 convertions opcodes
-* Reworked all callback mecanisms
-* Added libturbojpeg wrapping (and a hack for zoom to force using native one)
-* Added BOX86_SAFEMMAP env. var.
-* Reworked dynarec memory manager
-* Added a few opcodes
+* Fixes on some float to int x86 convertions opcodes.
```
[the remark this answers is not in the document]
Comment (heavygabriel) [reply CHANGELOG.md:22]: I have committed a fix. You can check it out now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,118 +1,118 @@
...
-* Improved the Init of dependant Libs. More things work now.
+* Improved the initialisation of dependent libraries. More things work now.
 * Added a lot of wrapped functions.
-* Added a few wrapped libs (like libz or some other x11 related libs)
-* For trace enabled build, Trace can be enabled only after a certain amount of opcodes (still, a debugger would be better)
-* Some fixes in a few opcodes, and implemented x87 rounding (SuperMeatBoy behaves better now)
-* FTL 1.6.9 still have corrupted music, but older 1.5.13 seems fine (different set of libs)
+* Added a few wrapped libraries (like libz or some other x11 related library).
+* For trace enabled build, Trace can be enabled only after a certain amount of opcodes (still, a debugger would be better).
+* Some fixes in a few opcodes, and implemented x86 rounding (SuperMeatBoy behaves better now).
```
[the remark this answers is not in the document]
Comment (heavygabriel) [reply CHANGELOG.md]: I have changed it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,118 +1,118 @@
...
 v0.0.2
 =======
-* A full commercial games runs fine on the Pandora: Airline Tycoon Deluxe
-* Implemented all planed subpart of Box86 except JIT
-* CPU Emulation is at 75%, roughly, including x87 and SSE/SSE2. MMX is barely implemented, but barely used anyway
-* x87 emulation is simplified, no real x87 State handling (but should not be mandatory, as native libm is used)
+* A full commercial games runs fine on the Pandora platform: Airline Tycoon Deluxe.
+* Implemented all planned subparts of Box86 except JIT support.
+* CPU Emulation is at roughly 75%. This includes x86 and SSE/SSE2. MMX is barely implemented (but barely used anyway).
+* x86 emulation is simplified, no real x86 State handling (but should not be mandatory, as the native libm library is used)
```
[the remark this answers is not in the document]
Comment (heavygabriel) [reply CHANGELOG.md]: Changed it.

File: USAGE.md [hunk 4]
```diff
@@ -7,120 +7,157 @@ There are many environment variables to control Box86 behaviour.
...
+Also, `/usr/lib/i386-linux-gnu` and `/lib/i386-linux-gnu` are added if they exist.
 
 #### BOX86_PATH
 Path to look for x86 executable. Default is current folder and `bin` in current folder.
 
 #### BOX86_DLSYM_ERROR
-* 0 : default. Don't log `dlsym` error
-* 1 : Log dlsym error
+Enables/Disables the logging of `dlsym` errors.
+ * 0
```
[the remark this answers is not in the document]
Comment (heavygabriel) [reply USAGE.md]: I have changed it, please let me know your feedback as I am unsure if I fixed it or not.

File: USAGE.md [hunk 5]
```diff
@@ -6,122 +6,118 @@ There are many environment variables to control Box86 behaviour.
...
- * 1
- * INFO : Show some minimum log (like librairies not found)
- * 2
- * DEBUG : Details a lot of stuff (like relocations or function called)
- * 3
- * DUMP : All DEBUG plus DUMP of all ELF Info
+ * 0: NONE : No message (except some fatal error). (Default.)
+ * 1: INFO : Show some minimum log (Example: librairies not found)
+ * 2: DEBUG : Details a lot of stuff (Example: relocations or functions called).
+ * 3: DUMP : All DEBUG plus DUMP of all ELF Info.
```
Comment (ptitSeb) [inline USAGE.md:12]: I don't think it's more clear.
Maybe * 0 / NONE; No message.... ?

Comment (ptitSeb) [reply USAGE.md:12]: Maybe on a next PR?

Comment (heavygabriel) [reply USAGE.md:12]: Yeah, it's definitely something to look into.

I will play around with markdown and see how to make it clearer.

**Pull request merged**